src/antsibull/changelog.py

```diff
@@ -489,6 +498,133 @@ def __init__(
         self.collection_metadata = collection_metadata
 
 
+def _get_removal_entry(
+    collection: str,
+    removal: RemovalInformation,
+) -> tuple[ChangelogFragment, str] | None:
+    if removal.announce_version is None:
+        return None
+
+    sentences = []
+    link = ""
+    if removal.discussion:
+        link = f" (`{removal.discussion} <{removal.discussion}>`__)"
+
+    if removal.reason == "deprecated":
+        sentences.append(f"The ``{collection}`` collection has been deprecated.")
+        sentences.append(
+            f"It will be removed from Ansible {removal.major_version} if no one"
+            " starts maintaining it again before Ansible {removal.major_version}."
+        )
+        sentences.append(
+            "See `the removal process for details on how this works"
+            " <https://docs.ansible.com/ansible/devel/community/collection_contributors/"
+            "collection_package_removal.html#canceling-removal-of-an-unmaintained-collection"
+            f">`__{link}."
```

Comment:
What do you think about wording it like this here and elsewhere?

```suggestion
            "See the `Collections Removal Process"
            " <https://docs.ansible.com/ansible/devel/community/collection_contributors/"
            "collection_package_removal.html#canceling-removal-of-an-unmaintained-collection"
            f">`__{link} for more details."
```

Reply:
Hmm, that's no longer correct, since the link is not to the Collection Removal Process, but to the How To Cancel for that process.

Reply:
Hmm, I see. My main thing is that the hyperlink is too long — I think it should only cover `removal process`, not `details on how this works` — and that `details on how this works` is a little informal.

Reply:
How about:
```suggestion
            "See the `Collections Removal Process for unmaintained collections"
            " <https://docs.ansible.com/ansible/devel/community/collection_contributors/"
            "collection_package_removal.html#unmaintained-collections"
            f">`__ for more details{link}."
```

Reply:
d8bd8e174e1d75b77c99e745dc38191f9e431bb9

Reply:
Thanks. This also applies to some of the other messages, I think.

Reply:
c7eb028f0f32974431d163e7462871480595ef4a